src/meltano/cli/invoke.py

```diff
@@ -112,7 +112,10 @@ def invoke(
             print_var=print_var,
         )
     )
-    with tracker.with_contexts(cmd_ctx):
+
+    with tracker.with_contexts(
+        cmd_ctx, PluginsTrackingContext([(plugin, command_name)])
```

Comment:
@aaronsteers quick and dirty example of how to attach a plugins context to a generic command event.